xwiki-platform-core/xwiki-platform-like/xwiki-platform-like-api/src/main/resources/templates/likers.vm

```diff
@@ -17,106 +17,118 @@
-    $response.setContentType('application/json')
-    #set ($documentReference = $doc.documentReference)
-    ##==============================
-    ## Offset = item # at which to start displaying data
-    ##==============================
-    #set($offset = $numbertool.toNumber($request.get('offset')))
-    ## offset starts from 0 in velocity and 1 in javascript
-    #set($offset = $offset - 1)
-    #if($offset < 0)
-        #set($offset = 0)
-    #end
-    ##==================
-    ## Limit = # of items to display
-    ##==================
-    #set($limit = $numbertool.toNumber($request.get('limit')))
-    ##==========
-    ## Sort direction
-    ##==========
-    #set($order = "$!request.sort")
-    #if($order != '')
-        #set($orderDirection = "$!{request.get('dir').toLowerCase()}")
-        #if("$!orderDirection" != '' && "$!orderDirection" != 'asc')
-            #set($orderDirection = 'desc')
-        #end
-    #end
-    #set ($likeRecords = $services.like.getLikers($documentReference, $offset, $limit))
-    #set ($userRows = [])
-    #foreach($userReference in $likeRecords)
-        #set ($grayed = $xcontext.userReference == $userReference.reference)
-        #set ($userDoc = $xwiki.getDocument($userReference.reference))
-        #set ($userProperties = $services.user.getProperties($userReference))
-        #set ($userObject = $user.getObject('XWiki.XWikiUsers'))
-        #if ($services.security.authorization.hasAccess('view', $userDoc.documentReference))
-            #set ($row = {
-                'grayed': $grayed,
-                'doc_fullName': $userDoc.fullName,
-                'doc_wiki': $userDoc.wiki,
-                'doc_url': $userDoc.getURL(),
-                'doc_viewable': true,
-                'name': "#displayUserAliasWithAvatar($userDoc.documentReference $disabled)",
-                'first_name': $userProperties.firstName,
-                'last_name': $userProperties.lastName
-            })
-        #else
-            #set ($row = {
-                'doc_viewable': false,
-                'doc_fullName': 'obfuscated'
-            })
-        #end
-        #set ($discard = $userRows.add($row))
-    #end
-    ## ===
-    ## JSON
-    ## ===
-    #set ($newOffset = $offset + 1)
-    #set ($optLikesNumber = $services.like.getLikes($documentReference))
-    #if ($optLikesNumber.isPresent())
-      #set ($totalRows = $optLikesNumber.get())
+  #end
+  #set ($likeRecords = $services.like.getLikers($documentReference, $offset, $limit))
+  #set ($userRows = [])
+  #foreach($userReference in $likeRecords)
+    #set ($grayed = $xcontext.userReference == $userReference.reference)
+    #set ($userDoc = $xwiki.getDocument($userReference.reference))
+    #set ($userProperties = $services.user.getProperties($userReference))
+    #set ($userObject = $user.getObject('XWiki.XWikiUsers'))
+    #if ($services.security.authorization.hasAccess('view', $userDoc.documentReference))
+      #set ($row = {
+        'grayed': $grayed,
```

Comment:
Is this supported by LiveData?

Reply:
It is not, but in the case of this Livetable, the property is never used. I'll check if this is caused by another regression.